src/devices/tuya.ts

```diff
@@ -1576,6 +1576,7 @@ const definitions: DefinitionWithExtend[] = [
             {vendor: 'Zemismart', model: 'LXZB-ZB-09A', description: 'Zemismart LED Surface Mounted Downlight 9W RGBW'},
             {vendor: 'Feconn', model: 'FE-GU10-5W', description: 'Zigbee GU10 5W smart bulb'},
             {vendor: 'Nedis', model: 'ZBLC1E14'},
+            {vendor: 'TechToy ', model: 'TS0505B', description: 'TechToy Smart Bulb RGB 9W E27 ZigBee'},
```

Comment:
Does this device have a model number written on the product page or site? Also can you use `tuya.whitelabel` like used below.

Reply:
It does not have anything on the product page. It literally took me a few hours to figure out that's it's a generic Tuya bulb >.<

Reply:
Resolved with 06ef8b60a2ecb28edc1c79eb3fcd86b0a1390d36